Patches/Mechanized Armor Set/Ammo_&_Apparel_Defs.xml

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Mechanized Armor Set</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				
+				<!--Plate Helm-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>
+						<WornBulk>0</WornBulk>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>3.5</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>5.2</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				
+				<!--Plate Armour-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>

```

Comment:
These should definitely have *some* worn bulk. Probably 10 - 15 would be appropriate. For reference, Recon Armor has 80 Bulk and 10 WornBulk. If you want to have some compensation for the high bulk, you can give is a bit of added weight carrying capacity--perhaps, 10 - 15 (recon armor gives 50, but also weighs more).